gsuite/config.go

```diff
@@ -22,6 +22,7 @@ var defaultOauthScopes = []string{
 	directory.AdminDirectoryGroupScope,
 	directory.AdminDirectoryUserScope,
 	directory.AdminDirectoryUserschemaScope,
+	directory.AdminDirectoryDomainScope,
```

Comment:
Setting this default may break existing users that don't expect the role to need the `DomainScope`. I'd rather have us document how to make sure it works but not set it as default (for now).

Reply:
Agree. Done.